src/pages/waiting-room-to-car/cat-a-mod1/waiting-room-to-car.cat-a-mod1.page.html

```diff
@@ -46,10 +45,3 @@
     </form>
   </div>
 </ion-content>
-<ion-footer [hidden]="pageState.eyesightTestFailed$ | async">
-  <ion-row class="mes-full-width-card box-shadow">
-    <button type="submit" class="mes-primary-button" id="continue-to-test-report-button" ion-button (click)="onSubmit()">
-      <h3>Continue to test report</h3>
-    </button>
-  </ion-row>
-</ion-footer>
```

Comment:
@edwardwilson how come?

Reply:
Eyesight isn't required in MOD1, saw it while updating schema changes